docs/chapter_tree/binary_tree.md

```diff
@@ -126,7 +126,12 @@
 === "Rust"
 
     ```rust title=""
-
+    /* 二叉树节点结构体 */
+    struct Node<T> {
+        value: T,
+        left: Option<Box<Node<T>>>,
```

Comment:
The definition of tree node should be like this:
```rust
use std::cell::RefCell;
use std::rc::Rc;

/* 二叉树节点结构体 */
struct TreeNode {
    val: i32,
    left: Option<Rc<RefCell<TreeNode>>>,
    right: Option<Rc<RefCell<TreeNode>>>,
}
```
You can find it in the  [tree_node.rs](https://github.com/krahets/hello-algo/blob/main/codes/rust/include/tree_node.rs).

And an instantiation method needs to be added:
```rust
impl TreeNode {
    /* 节点初始化方法 */
    fn new(val: i32) -> Self {
        Self { val, left: None, right: None }
    }
}
```